plugins/mcreator-core/enchantmenteffects/ench_component_prevent_armor_change.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extensions": [
+    "disable_repeated_enchantment_component"
+  ],
+  "inputsInline": true,
+  "previousStatement": "EnchantmentComponent",

```

Comment:
This is new one to me. So this defines "type" of block blocks can connect to from what I see.

But how is the type of the current block defined? Like how do other blocks know this one is also EnchantmentComponent?

Reply:
Thanks for the explanation!